src/hooks/useBridgeClaimNotification/useBridgeClaimNotification.tsx

```diff
@@ -0,0 +1,69 @@
+import { Alert, AlertDescription, Button, CloseButton, Flex, useToast } from '@chakra-ui/react'
+import type { ResponsiveValue } from '@chakra-ui/system'
+import type { Property } from 'csstype'
+import { useEffect, useState } from 'react'
+import { FaInfoCircle } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { IconCircle } from 'components/IconCircle'
+import { useArbitrumClaimsByStatus } from 'components/MultiHopTrade/components/TradeInput/components/Claim/hooks/useArbitrumClaimsByStatus'
+import { TradeRoutePaths } from 'components/MultiHopTrade/types'
+
+const flexGap = { base: 2, md: 3 }
+const flexDir: ResponsiveValue<Property.FlexDirection> = { base: 'column', md: 'row' }
+const flexAlignItems = { base: 'flex-start', md: 'center' }
+
+export const useBridgeClaimNotification = () => {
+  const toast = useToast()
+  const history = useHistory()
+  const translate = useTranslate()
+  const [isDisabled, setIsDisabled] = useState(false)
+
+  const { claimsByStatus, isLoading } = useArbitrumClaimsByStatus({ isDisabled })
+
+  useEffect(() => {
+    if (isLoading || isDisabled) return
+    if (claimsByStatus.Available.length === 0) return
+
+    // trigger a toast
+    toast({
+      render: ({ onClose }) => {
+        const handleCtaClick = () => {
+          history.push(TradeRoutePaths.Claim)
+          onClose()
+        }
+
+        return (
+          <Alert status='info' variant='update-box' borderRadius='lg' gap={3}>
+            <IconCircle boxSize={8} color='text.subtle'>
+              <FaInfoCircle />
+            </IconCircle>
+            <Flex gap={flexGap} flexDir={flexDir} alignItems={flexAlignItems}>
+              <AlertDescription letterSpacing='0.02em'>
+                {translate('bridge.availableClaimsNotification')}
+              </AlertDescription>
+
+              <Button
+                colorScheme='blue'
+                size='sm'
+                // translate and history undefined if this component split out, so cannot properly memoize
+                // eslint-disable-next-line react-memo/require-usememo
+                onClick={handleCtaClick}
+              >
+                {translate('bridge.viewClaims')}
+              </Button>
+            </Flex>
+            <CloseButton onClick={onClose} size='sm' />
+          </Alert>
+        )
+      },
+      id: 'bridge-claim',
+      duration: null,
+      isClosable: true,
+      position: 'bottom-right',
+    })
+
+    // don't spam user
```

Comment:
Two qs: 
- if the only intent is to stop the query from refiring, can we use infinite `staletime` / `gcTime`?
- Probably a nitpick but will it stop claims detection when switching from one wallet to another after this was set to `true`?

Reply:
1. Good idea, I can do that
2. Probably, I'll add a useEffect to reset it when the wallet ID changes

Reply:
1. a690042b38857005977129f21a29dd96db02159a
2. b9d9b8ec724dce76a37c9f82d7921fa91ba88b2c